test/Incremental/Verifier/single-file-private/AnyObject.swift

```diff
@@ -25,7 +25,6 @@ import Foundation
 // expected-private-conformance {{Swift.CVarArg}}
 // expected-private-conformance {{Swift.CustomStringConvertible}}
 // expected-private-member {{Swift._ExpressibleByBuiltinIntegerLiteral.init}}
-// expected-private-superclass {{main.LookupFactory}}
```

Comment:
Is this expected?

Reply:
Yeah, we're no longer doing conformance lookup for a specific set of conformances. Rather, we ask for which conformances are ascribed to this particular decl context, and force those if they are derivable.

Reply:
I'm surprised to see these edges go, but it shouldn't affect anything in practice. The cross-file tests losing this dependency would be the catastrophic one.